### ISSUE TITLE
CSV Dataset support

### ISSUE THREAD

**Issue by Minitour** [eldaduzman/pymeter#82]
Consider porting the CSV DataSet so tests can be carried out using variables from a list and not only random/hardcoded values.

https://github.com/abstracta/jmeter-java-dsl/blob/master/jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/JmeterDsl.java#L1359:1361

**Comment by Minitour** [comment]
My current workaround is making my own class:
```python
from pymeter.api import  TestPlanChildElement


class CsvDataset(TestPlanChildElement):
    def __init__(self, name: str) -> None:
        self._csv_dataset_instance = TestPlanChildElement.jmeter_class.csvDataSet(name)
        super().__init__()

```

**Comment by eldaduzman** [comment]
Released in version 0.4
